src/components/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Grid, Paper, Container } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+
+const styles = (theme) => ({
+  content: {
+    flexGrow: 1,
+  },
+  description: {
+    padding: theme.spacing(2),
```

Comment:
The left padding from this style makes it look unaligned. try using padding with 4 values to make the left value different. This would work ```padding: `${theme.spacing(2)}px ${theme.spacing(2)}px ${theme.spacing(2)}px 0px`,```. This is similar to padding: '12px 12px 12px 0px' but since we are using a variable, we use template literals instead of single quote.